assets/css/styles.scss

```diff
@@ -3,8 +3,8 @@
 
 // Declare variables that you need from _config.yml here
 // Agency brand colors
-$primary-color: {{- site.colors.primary-color -}};
-$secondary-color: {{- site.colors.secondary-color -}};
+$primary-color: {{- site.colors.primary-color | default: '#f5a623' -}};
+$secondary-color: {{- site.colors.secondary-color | default: '#50e3c2' -}};
```

Comment:
i don't disagree with this change - however, we need to be clear that this should be specified at build creation time (ie, when jekyll attempts to build the site). 

what this does is provide a sensible default (which is fine) but it obscures the source of error (which is **not** fine). i'm more in favour of removing it here and guaranteeing it exists at point of site creation

Reply:
Removed! It's safe to remove it in the template now since the configuration is now set inside `_config.yml`, so no errors are thrown when building.